README.md

```diff
@@ -117,3 +117,16 @@ issues created.
 
 Details on the request and expected response are [available](docs/generic\_webhook.md).
 
+
+## Cleanup script
+
+A python script is provided in scripts/cleanup.py and in a docker container. It can be configured
+using the commandline options to filter rageshakes in a given date range. It is expected to be
+run as a cronjob on a daily basis.
+
+The script is idempotent enough to be re-runnable on a more frequent basis if required. If a cronjob
+cannot be configured, it can be run in a continual mode with something like `--repeat-delay-hours 24`
+to repeat running after approximately 24 hours.
```

Comment:
Is there any particular reason to prefer a cronjob?

```suggestion
using the commandline options to filter rageshakes in a given date range.

It can either be run via a cronjob at appropriate intervals (typically daily), or
be set to run in a continual mode with something like `--repeat-delay-hours 24`
to repeat running after approximately 24 hours.
```